go.mod

```diff
@@ -16,7 +16,10 @@ require (
 	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/viper v1.7.0
 	github.com/urfave/cli/v2 v2.3.0
-	golang.org/x/crypto v0.0.0-20201216223049-8b5274cf687f // indirect
+	golang.org/x/crypto v0.0.0-20220411220226-7b82a4e95df4 // indirect
```

Comment:
This lib seems not existing in https://github.com/apache/skywalking-cli/blob/master/dist/LICENSE

Reply:
This is not the only missing case, we have lots of indirect dependencies missing in the dist/LICENSE file and that needs an overhaul before next release.